### DISCUSSION TITLE
Contributing datasets

### DISCUSSION THREAD

**Discussion by ssagawa** [p-lambda/wilds#6] [General]
Hello everyone,

We are actively adding datasets to WILDS, and we would highly welcome community contributions. If you have datasets that you think would be appropriate, please let us know! 

We are specifically searching for datasets with **subpopulation shifts** as well as datasets that could be used to study algorithmic fairness. Some potential examples include speech datasets (where the subpopulations might be different speaker backgrounds/accents) or medical datasets (where the subpopulations might be different pathology subclasses or patient demographics). We describe other potential applications and datasets in Section 6 of our [paper](https://arxiv.org/pdf/2012.07421.pdf). If you have any promising leads, please post here or contact us directly at wilds@cs.stanford.edu.

Thank you!
WILDS Team

**Comment by kohpangwei** [comment]
In case it's helpful for people looking into this, [Robustness Gym](https://github.com/robustness-gym/robustness-gym) (Goel et al., 2021) has nice built-in support for evaluating on different subpopulations, as well as other forms of robust evaluations.

**Comment by naymaraq** [comment]
Hi @ssagawa, WILDS Team

I recently created a speaker recognition dataset for domain generalization. I am wondering how I can contribute to WILDS?

**Problem setting**
The task is to predict if two audio files contain speech from the same speaker, where the domains are languages. And our goal is to learn language-agnostic feature extractor. In this setting (unlike other WILDS tasks), the labels (speaker ids) are not shared across domains.

**Reply by kohpangwei** [reply]
Hello! Thanks for reaching out. With the release of v2.0 earlier this year, we're not currently actively looking for datasets to add to WILDS. But if you have a paper that describes the dataset, please let us know and we'd be happy to take a look. Thank you.
